download_jartic_archives.py

```
@@ -0,0 +1,127 @@
+    
+    print(f"\n{'='*60}")
+    print("DOWNLOAD SUMMARY")
+    print(f"Total archives: {len(months_to_download)}")
+    print(f"Downloaded: {downloaded_count}")
+    print(f"Cached: {cached_count}")
+    print(f"Failed: {failed_count}")
+    print(f"Cache directory: {cache_dir}")
+    print(f"{'='*60}")
+
+
+def parse_date(date_str):
```

Ruff (0.12.2)

Missing return type annotation for public function `parse_date`

(ANN201)